src/openapi.yaml

```
@@ -869,6 +869,52 @@ paths:
           headers:
             X-Session-Expires-In:
               $ref: '#/components/headers/X-Session-Expires-In'
+  /api/v1/forms/{form_uuid_or_slug}/variables/bulk_update:
```

```suggestion
  /api/v1/forms/{form_uuid_or_slug}/variables:
```

would probably be a nicer endpoint capturing the resource semantics?

---

I am not entirely sure how to go about this (because it's a nested endpoint) :thinking: